jobs/queue.go

```diff
@@ -0,0 +1,125 @@
+package jobs
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/go-redis/redis"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+)
+
+type Queue struct {
+	Name       string
+	BackupName string
+	Window     int64
```

Comment:
This is in seconds?

Reply:
yeah it was a copy pasta - i might rework some of that but for now i think its okay